pontoon/actionlog/models.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+
+class ActionLog(models.Model):
+    ACTIONS_TYPES = (
+        # A translation has been created.
+        ('translation:created', 'Translation created'),
+
+        # A translation has been deleted.
+        ('translation:deleted', 'Translation deleted'),
```

Comment:
[question] @adngdb what do you think about using enums in Python?
An enum with all possible actions could minimize some of the problems of "string-based" etc. (sorry if I'm stating the obvious)

Additionally, https://pypi.org/project/django-enum-choices/ gives you helpers to generate a choice field based on an enum-like class.

Reply:
I'm unsure about this. On one hand, I recognize that it's better to use an enum for this use case. On the other, it means more `import`s, and we already have this behavior coded with the type validation on `save`. 

Given that `enum` has been added in py3.4 and we'd need to add one more dependency and rewrite a bunch of code, I think I'd rather not do it. But if anyone has strong opinions in favor of enums, I'm happy to read them.